src/gardenlinux/features/cname.py

```diff
@@ -395,9 +429,14 @@ def load_from_release_file(self, release_file: PathLike | str) -> None:
             UNNAMED_SECTION, "GARDENLINUX_FEATURES_FLAGS"
         ).split(",")
 
-        self._feature_platforms_cached = release_config.get(
-            UNNAMED_SECTION, "GARDENLINUX_FEATURES_PLATFORMS"
-        ).split(",")
+        self._feature_platform_cached = release_config.get(
+            UNNAMED_SECTION, "GARDENLINUX_FEATURES_PLATFORM"
```

Comment:
This is not exactly how the new platform logic works. See: https://github.com/gardenlinux/gardenlinux/blob/main/features/base/exec.config#L21-L26

It should use the `GARDENLINUX_FEATURES_PLATFORMS` for the set of all platforms (so this still is an array if one forces the builder to accept multiple platforms via `--allow-frankenstein`) and `GARDENLINUX_PLATFORM` for the one and only platform, which will instead read `frankenstein` to indicate no clear single one and only platform can be determined.

Reply:
Differentiation integrated